lxd/include/memory_utils.h

```diff
@@ -1,22 +1,5 @@
-/* liblxcapi
- *
- * Copyright © 2019 Christian Brauner <christian.brauner@ubuntu.com>.
- * Copyright © 2019 Canonical Ltd.
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
-
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
-
- * You should have received a copy of the GNU Lesser General Public License
- * along with this library; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
- */
+// SPDX-License-Identifier: LGPL-2.1+
+/* Copyright © 2019 Christian Brauner <christian.brauner@ubuntu.com>. */
```

Comment:
This is a problem. LXD is Apache2 and we're not dynamically linking this stuff in, so we need to re-license that bit of code to Apache2 in the context of LXD.

Reply:
We're currently shipping that headeunder LGPL...